modules/ossm-config-sampling.adoc

```diff
@@ -5,16 +5,16 @@
 [id="ossm-config-sampling_{context}"]
 = Adjusting the sampling rate
 
-The distributed tracing sampling rate is set to sample 100% of traces in your service mesh by default. A high sampling rate consumes cluster resources and performance but is useful when debugging issues. Before you deploy {ProductName} in production, set the value to a smaller proportion of traces.
+The Envoy sampling rate is set to sample 100% of traces in your service mesh by default. A high sampling rate consumes cluster resources and performance but is useful when debugging issues. Before you deploy {ProductName} in production, set the value to a smaller proportion of traces.
```

Comment:
Could you say, `The Envoy sampling rate samples 100% of traces in your service mesh by default`? If you think this is too much anthropomorphism, feel free to keep as it is. Thought we might be able to easily make it active :)

Reply:
It would be more correct to say that "By default, Envoy is configured to sample 100% of traces in your service mesh."  but that's still passive voice.    ¯\_(ツ)_/¯